analyzer/src/main/kotlin/managers/PackageJsonUtil.kt

```diff
@@ -37,7 +37,7 @@ internal class PackageJsonUtil {
                 val hasYarnLockfile: Boolean = false,
                 val hasNpmLockfile: Boolean = false,
                 val isYarnWorkspaceRoot: Boolean = false,
-                var isYarnWorkspaceSubmodule: Boolean = false
+                val isYarnWorkspaceSubmodule: Boolean = false
```

Comment:
Commit message nit: In the previous commit you say "properties", here you say "parameter". Please align on one wording. Just FYI, personally I'd also be fine with squashing this to the previous commit.

Reply:
Done, without squashing.